### PR TITLE
enh(style): Use eslint-config-centreon and fix ESLint errors

### PR DESCRIPTION
**Pull request by bdauria**
#Ref https://centreon.atlassian.net/browse/MON-3898

I ran eslint --fix to automatically fix issues, and disabled errors locally within files to be conservative.

I tested the Components using storybook, I still need to test using centreon-web and centreon-bam, as well as launching the Jenkins builds.

### CODE REVIEW

**Comment by danielhalupka** [comment]
Please fix the conflicts

**Comment by bdauria** [comment]
@danielhalupka master merged & conflicts resolved.

**Comment by bdauria** [comment]
Build status for centreon-web: https://jenkins.centreon.com/job/centreon-web/job/mon-3898-fix-eslint-errors/

File: src/Dialog/Confirmation/index.js [hunk 1]
```diff
@@ -36,14 +46,14 @@ class ConfirmationDialog extends Component {
             variant="contained"
             color="primary"
             style={{
```
Comment (EditApp) [inline src/Dialog/Confirmation/index.js]: In the future please thing to refactoall inline styles.

**Pull request closed without merging**